src/kOS/Module/kOSProcessor.cs

```diff
@@ -82,7 +95,7 @@ public void Activate()
             OpenWindow();
         }
 
-        [KSPField(isPersistant = true, guiName = "Required Power", guiActive = true)]
+        [KSPField(isPersistant = true, guiName = "kOS Required Power", guiActive = true, guiActiveEditor = true)]
```

Comment:
Are there any means of having tooltips on these fields?  It would be nice if we could explain that this is the power if and only if the cpu is running at 100% and not doing any waits.